changes/tasks.py

```diff
@@ -524,6 +560,17 @@ def check_momconnect_change_language(self, data_fields, change):
         else:
             return []
 
+    def check_momconnect_change_msisdn(self, data_fields, change):
+        """
+        Ensures that a new msisdn is provided, and that it is a valid msisdn.
+        """
+        if "msisdn" not in data_fields:
```

Comment:
So that we can remove the numbers from the change object when the identity gets changed? I guess that works.

It isn't an option for other field changes though, like id numbers, since those don't have detail blobs.